usecases/config/config_handler.go

```diff
@@ -73,6 +73,7 @@ type Config struct {
 	Debug                               bool                     `json:"debug" yaml:"debug"`
 	QueryDefaults                       QueryDefaults            `json:"query_defaults" yaml:"query_defaults"`
 	QueryMaximumResults                 int64                    `json:"query_maximum_results" yaml:"query_maximum_results"`
+	QueryNestedRefLimit                 int64                    `json:"query_ref_limit" yaml:"query_ref_limit"`
```

Comment:
let's make the json / yaml variable name the same as `QUERY_NESTED_CROSS_REFERENCE_LIMIT` environment variable name for example: `query_nested_cross_reference_limit`

Reply:
fixing